src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -118,4 +131,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         $io->newLine();
     }
+
+    private function getFileLink(string $class)
+    {
+        try {
+            $r = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return '';
+        }
+
+        return sprintf('phpstorm://open?file=%s&line=%d', $r->getFileName(), $r->getStartLine());
```

Comment:
Not everyone is using phpstorm: there is a service that generates the links for you, look for `debug.file_link_formatter`/`FileLinkFormatter` elsewhere.

Reply:
True, was looking for a solution to fix this, Thanks @nicolas-grekas